openreview/venue_request/process/matchingProcess.py

```diff
@@ -23,62 +23,62 @@ def process(client, note, invitation):
         matching_status = conference.setup_committee_matching(matching_group, compute_affinity_scores, compute_conflicts)
     except Exception as e:
         if 'Submissions not found.' in str(e):
-            matching_status['error'] = ['Could not compute affinity scores and conflicts since no submissions were found. Make sure the submission deadline has passed and you have started the review stage using the \'Review Stage\' button.']
+            matching_status['error'] = 'Could not compute affinity scores and conflicts since no submissions were found. Make sure the submission deadline has passed and you have started the review stage using the \'Review Stage\' button.'
         elif 'The match group is empty' in str(e):
-            matching_status['error'] = [f'Could not compute affinity scores and conflicts since there are no {role_name}. You can use the \'Recruitment\' button to recruit {role_name}.']
+            matching_status['error'] = f'Could not compute affinity scores and conflicts since there are no {role_name}. You can use the \'Recruitment\' button to recruit {role_name}.'
         elif 'The alternate match group is empty' in str(e):
             role_name = conference.get_area_chairs_name()
-            matching_status['error'] = [f'Could not compute affinity scores and conflicts since there are no {role_name}. You can use the \'Recruitment\' button to recruit {role_name}.']
+            matching_status['error'] = f'Could not compute affinity scores and conflicts since there are no {role_name}. You can use the \'Recruitment\' button to recruit {role_name}.'
         else:
-            matching_status['error'] = [str(e)]
+            matching_status['error'] = str(e)
 
     print("Following error in the process function was posted as a comment:")
     print(traceback.format_exc())
 
     comment_note = openreview.Note(
-        invitation = note.invitation.replace('Paper_Matching_Setup', 'Comment'),
+        invitation = note.invitation.replace('Paper_Matching_Setup', 'Paper_Matching_Setup_Status'),
         forum = note.forum,
         replyto = note.id,
         readers = [conference.get_program_chairs_id()] + [SUPPORT_GROUP],
         writers = [],
         signatures = [SUPPORT_GROUP],
         content = {
-            'title': 'Matching Status [{note_id}]'.format(note_id=note.id),
+            'title': 'Paper Matching Setup Status',
             'comment': ''
         }
     )
 
     if matching_status.get('error'):
-        error_status = f'''{len(matching_status.get('error'))} error(s): {matching_status.get('error')}'''
-        comment_note.content['comment'] += error_status
-        comment_note.content['comment'] += f'''
-
-To check references for the note: https://api.openreview.net/references?id={note.id}
-'''
+        error_status = f'''{len(matching_status.get('error'))} error(s): \n
```

Comment:
Looks like  `{len(matching_status.get('error'))}` here is just getting the lenght of the error string? It shouldn't say there are 136 error(s) (and the error string has 136 characters)

![Screen Shot 2022-04-01 at 2 09 52 PM](https://user-images.githubusercontent.com/32438984/161319074-e0c6e8b0-ab6a-4fdb-ab8f-134dc1e0049b.png)

Reply:
Removed the count of errors as it was always 1.